powerauth-push-server/src/main/java/io/getlime/push/controller/rest/InboxController.java

```diff
@@ -52,17 +52,39 @@ public class InboxController {
 
     private final InboxService inboxService;
 
+    /**
+     * Default constructor of the controller with inbox service injection.
```

Comment:
Nitpick: this is not _default_ but _all arg_ constructor. You may skip explicit constructor and Javadoc by a lombok annotation `RequiredArgsConstructor`.